Auth0Tests/AuthenticationSpec.swift

```diff
@@ -496,7 +499,8 @@ class AuthenticationSpec: QuickSpec {
                                    challenge: signupChallenge,
                                    connection: ConnectionName,
                                    audience: Audience,
-                                   scope: Scope)
+                                   scope: Scope,
+                                   organization: OrganizationId)
```

Comment:
Same here: the value of `OrganizationId` should be included in the `hasAtLeast` call, to check that the SDK is actually sending the param.